demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -461,33 +461,25 @@ def report_failed_lint_checks(self, lint_status: dict, pkgs_status: dict, return
             pkgs_status(dict): All pkgs status dict
             return_exit_code(int): exit code will indicate which lint or test failed
         """
-        for check in ["flake8", "XSOAR_linter", "bandit", "mypy", "vulture"]:
+        for check in ["flake8", "XSOAR_linter", "bandit", "mypy", "vulture", "pylint", "pwsh_analyze", "pwsh_test"]:
```

Comment:
1. Let's make this a constant as well.
2. What about pytest?

Reply:
there is dedicated method for test reports 
`report_unit_tests`